shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -172,27 +172,27 @@ FLUTTER_EXPORT
  * it will signal the engine to stop animations and drawing.  However, neither will impact the state
  * of the Dart program's execution.
  */
-@property(nonatomic, weak) FlutterViewController* viewController;
+@property(nonatomic, weak, nullable) FlutterViewController* viewController;
 
 /**
  * The `FlutterMethodChannel` used for localization related platform messages, such as
  * setting the locale.
  */
-@property(nonatomic, readonly) FlutterMethodChannel* localizationChannel;
+@property(nonatomic, readonly, nonnull) FlutterMethodChannel* localizationChannel;
```

Comment:
This can be null

Reply:
this would be only after destroyContext right? Will change and update doc.

Reply:
Right now yes.